### PR TITLE
Fetch product schema from API

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Add fetch logic inside FromTemplateService.

Remove JSON schemas from data-folder.

Add new query to fetch product.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #563** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/563" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/563?utm_source=stack-comment).

**Review by simonauner — APPROVED** [review]
LGTM!

gif

File: apps/store/src/services/formTemplate/FormTemplateService.ts [hunk 1]
```diff
@@ -46,4 +38,20 @@ export class FormTemplateService {
       ...combineFormTemplate({ schema, uiSchema }),
     }
   }
+
+  private async fetchSchema(productName: string) {
+    try {
+      const result = await this.apolloClient.query<ProductQuery, ProductQueryVariables>({
+        query: ProductDocument,
+        variables: { name: productName },
+        fetchPolicy: 'no-cache',
```
Comment (alebedev) [inline apps/store/src/services/formTemplate/FormTemplateService.ts:47]: Why do we need it?

**Review by alebedev — APPROVED** [review]
Looks good!

**Pull request merged**